runtime/doc/nvim_terminal_emulator.txt

```diff
@@ -5,6 +5,7 @@
 
 
 Terminal emulator					   *terminal-emulator*
+							     *terminal-buffer*
```

Comment:
I changed this to just `*terminal*`. In a pending PR I cleaned up all user docs so that the word "terminal" is always understand to mean the "built-in embedded terminal", and "TUI" is used in the few cases where we need to discuss external terminals.